README.md

```diff
@@ -1 +1,83 @@
-# Awesome-Books
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+> **[Awesome Books]** is a simple website that displays a list of books and allows you to add and remove books from that list
+
+## 🛠 Built With <a name="built-with"></a>
+
+> HTML,
+> CSS and
+> JavaScripts
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://mohamedhnoor.github.io/Awesome-Books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Prerequisites
+
+In order to run this project you need:
+
+> Computer with internet and
+> Visual Studio Code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohamed H Noor**
+
+- GitHub: [@githubhandle](https://github.com/MohamedHNoor)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamedhnoor/)
+
+👤 **Christian Ani**
+
+- GitHub: [@githubhandle](https://github.com/iamchristianani)
+- Twitter: [@kriznode]((https://twitter.com/kriznode)
+- LinkedIn: [Christian Ani](https://www.linkedin.com/in/anikriz/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/MohamedHNoor/Awesome-Books/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please give a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank my coding partner
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
-  Welldone making your readme file descriptive, kindly ensure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) are present, for example the Tech stack and future projects section are missing please add it 👍🏿  👍🏿